utils/pom.xml

```diff
@@ -261,7 +261,7 @@
                             <goal>shade</goal>
                         </goals>
                         <configuration>
-                            <finalName>${project.artifactId}-${project.version}-SHADED</finalName>
+                            <finalName>${project.artifactId}-${project.version}-SHADED-2</finalName>
```

Comment:
can you advise what's causing the failures @weizhouapache - should we call it something else like `bundled` or `shaded` lowercase?

Reply:
@rohityadavcloud 
maven will create `-shaded.jar` and copy to `-SHAED.jar`. it works on Linux.
I have no idea why it fails on MacOS.

`bundled` looks good. I will rename it.